2200028/3.py

```diff
@@ -0,0 +1,16 @@
+T = int(input())
+
+for t in range(1, T+1):
+    list = input()
+    list = list[::-1]
+    result = []
```

Comment:
조건을 나눠서 잘 풀어주셨네요! 😄 
```suggestion
    result = ''
```
`result`의 초기 값을 string 으로 설정해주셔야 합니다!